library/ssl_tls12_client.c

```diff
@@ -1656,6 +1661,15 @@ static int ssl_parse_server_hello(mbedtls_ssl_context *ssl)
         return MBEDTLS_ERR_SSL_HANDSHAKE_FAILURE;
     }
 
+#if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
+    if (ssl->session_negotiate->mfl_code != MBEDTLS_SSL_MAX_FRAG_LEN_NONE && \
```

Comment:
(minor) don't need the continuation character `\` - this is C code, not the pre-processor

Reply:
yeah, true. removed.